web/src/beta/lib/core/engines/Cesium/Sketch/ExtrudedPolygonEntity.tsx

```diff
@@ -53,12 +55,18 @@ export const ExtrudedPolygonEntity: FC<ExtrudedPolygonEntityProps> = ({
         extrudedHeight,
         extrudedHeightReference: HeightReference.RELATIVE_TO_GROUND,
         fill: true,
+        outline: true,
+        outlineWidth: 1,
```

Comment:
Should the outline be customizable from plugin API?

Reply:
For the sketch API i want to have a refactor in another PR. So i'll skip this one here.